Move update for multiple support in Select to a patch

cbrandtbuffalo opened the pull request:
Hi Cloos,

I noticed the dropdowns don't render with bootstrap styling in RT 5, so my suggestion is to move the change to a patch since only really old RTs don't have that change now.

Thanks,
Jim

cbrandtbuffalo committed:
Provide support for multiple select as a patch for older RTs

cbrandtbuffalo committed:
Remove the Select overlay template in favor of a patch

RT 5 switches to bootstrap, so the Select widget is now different
between RT 4 and RT 5. This makes it harder to provide an overlay
that will work on both.

Multiple support has been in core RT for a long time, so move
the update that added it to a patch for continuity on older
RTs and remove the overlay.

cbrandtbuffalo committed:
Remove the Select overlay if needed on upgrade

netsandbox commented:
@cbrandtbuffalo I don't understand the purpose of the `remove_files`option.
If I test your PR, `make install` shows:
```
No remove file located, no files to remove
```
and  `/opt/rt5/local/plugins/RT-Extension-HistoryFilter/html/Widgets/Form/Select` is still there.

cbrandtbuffalo committed:
Push dot (.) into INC just for the install process

The current directory is still needed for some installation
tools, so add dot(.) to INC to install correctly on Perl
5.26 and later.

cbrandtbuffalo commented:
It's supposed to remove the old file from the installed RT, but it expects . to be in INC, so it wouldn't find the list of files to remove on perl 5.26 or later. I pushed another commit to add . to INC in Makefile.PL just for the installer. Sorry about that, could you give it a try now?
